mage_integrations/mage_integrations/sources/pipedrive/__init__.py

```diff
@@ -0,0 +1,44 @@
+from singer import catalog as catalog_singer
+import json
+
+from mage_integrations.sources.base import Source, main
+from mage_integrations.sources.catalog import Catalog
+from mage_integrations.sources.pipedrive.tap_pipedrive.tap import PipedriveTap
+
+from mage_integrations.utils.dictionary import ignore_keys
+from typing import List
+
+
+class Pipedrive(Source):
+    def discover(self, streams: List[str] = None) -> Catalog:
+        pipedrive_tap = PipedriveTap(json.loads(json.dumps(self.config)),
```

Comment:
I was receiving single quote json from self.config. So i figured the most pythonic way to deserialize it to python structure was the json.loads(json.dumps()) strategy ( since this tap configures args in singer parse_args format, to run it on CLI)

Reply:
I think they're the same. `self.config` should be a dictionary already. Could you test just using `self.config` directly?

Reply:
Tested it and it worked as expected. Removed json.loads(json.dumps)) on commit 3d8fd0c2e555279c6849a39f59c5b6b70f3afd93